cabal.project

```diff
@@ -14,7 +14,7 @@ repository cardano-haskell-packages
 -- you need to run if you change them
 index-state:
   , hackage.haskell.org 2024-10-11T15:49:11Z
```

Comment:
I didn't update hackage, because I ran into a number of problems, because our transitive dependencies (ledger, plutus) use an old hackage. We will be able to get a more recent hackage when ledger does.